crates/fluvio-protocol/Cargo.toml

```diff
@@ -61,7 +61,7 @@ tracing = { workspace = true }
 fluvio-protocol-derive = { version = "0.5.0", path = "../fluvio-protocol-derive", optional = true }
 fluvio-future = { workspace = true, optional = true }
 flv-util = { workspace = true,  optional = true }
-fluvio-compression = { path = "../fluvio-compression", version = "0.3.0", default-features = false, optional = true }
+fluvio-compression = { path = "../fluvio-compression", version = "0.3.0", default-features = false, features = ["compress"], optional = true }
```

Comment:
seems like unnecessary feature flag.  compression need to compress

Reply:
I was keeping it to reduce amount of changes, will give a shot on removing it!